textscreen/txt_sdl.c

```diff
@@ -162,14 +162,32 @@ int TXT_Init(void)
         return 0;
     }
 
+    SDL_RendererLogicalPresentation mode = SDL_LOGICAL_PRESENTATION_LETTERBOX;
+
+    if (window_type == TXT_WINDOW_ENDOOM)
+    {
+        int h, border_h;
+
+        SDL_GetWindowSizeInPixels(TXT_SDLWindow, NULL, &h);
+        border_h = h % screen_image_h;
+
+        if (border_h >= 0 && border_h <= h / 4)
```

Comment:
Perhaps 25% is too large? Our default window size is 600p, so ENDOOM appears too small. What about 20% (`border_h <= h / 5`)?

Reply:
Sure, here's the new breakdown:

<img width="371" height="372" alt="border" src="https://github.com/user-attachments/assets/890b2d59-2273-43f5-847a-5b2d9f170ce8" />